mayastor/src/lvs/lvol.rs

```diff
@@ -239,18 +279,18 @@ impl Lvol {
     }
 
     /// destroy the lvol
-    #[instrument(level = "debug", err)]
     pub async fn destroy(self) -> Result<String, Error> {
         extern "C" fn destroy_cb(sender: *mut c_void, errno: i32) {
             let sender =
                 unsafe { Box::from_raw(sender as *mut oneshot::Sender<i32>) };
             sender.send(errno).unwrap();
         }
 
-        let name = self.name();
-
         // we must always unshare before destroying bdev
         let _ = self.unshare().await;
+        self.wipe_super().await?;
```

Comment:
In case we can't wipe out the superblock (due to I/O errors on superblock or so), should that block us from removing the underlaying LVOL ?

Reply:
Yeah I thought about that too, we could swallow the error but IMHO, it might hide some other issues if we do that. We can always think about it later IMHO.

Reply:
No problem, at least we log all the errors. LGTM.